### ISSUE TITLE
assert config.vision_config.model_type == 'davit', 'only DaViT is supported for now'

### ISSUE THREAD

**Issue by wavelet2008** [andimarafioti/florence2-finetuning#4]
im encountering a new issue after fine-tuning model.

(facefl) ubuntu@ubuntu-X10SRA:~/sdf/florence/florence2-finetuning$ python evaluate.py 
Traceback (most recent call last):
  File "/home/ubuntu/sdf/florence/florence2-finetuning/evaluate.py", line 15, in <module>
    model = AutoModelForCausalLM.from_pretrained(
  File "/home/ubuntu/anaconda3/envs/facefl/lib/python3.10/site-packages/transformers/models/auto/auto_factory.py", line 558, in from_pretrained
    return model_class.from_pretrained(
  File "/home/ubuntu/anaconda3/envs/facefl/lib/python3.10/site-packages/transformers/modeling_utils.py", line 3626, in from_pretrained
    model = cls(config, *model_args, **model_kwargs)
  File "/home/ubuntu/.cache/huggingface/modules/transformers_modules/modeling_florence2.py", line 2532, in __init__
    assert config.vision_config.model_type == 'davit', 'only DaViT is supported for now'
AssertionError: only DaViT is supported for now

**Comment by m2key1** [comment]
@wavelet2008 I have encountered the same issue, did you manage to solve it?

**Comment by sylvain471** [comment]
Hello, 

I encountered this as well while loading the checkpoint to perform inference. 

I solved it by visiting the model repo at huggingface microsoft/Florence-2-base-ft (actually I used my local cache folder instead) and copying the config.json together with the scripts `configuration_florence2.py`, `modeling_florence2.py` and `processing_florence2.py`.

When save using `.save_oretrained` it seems that the scripts are not saved and the config.json is missing the following vision config attributes `"model_type": "davit"`, and probably more.

I also removed the `revision="refs/pr/6"`  bit from the model and processor imports.

I hope this helps
